firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -94,13 +94,15 @@ public void start() {
         .forEach(
             row -> {
               try {
+                int indexId = row.getInt(0);
+                String collectionGroup = row.getString(1);
+                List<FieldIndex.Segment> segments =
+                    serializer.decodeFieldIndexSegments(Index.parseFrom(row.getBlob(2)));
+                SnapshotVersion updateTime =
+                    new SnapshotVersion(new Timestamp(row.getLong(3), row.getInt(4)));
```

Comment:
Seconds are stored in SQLite as `INTEGER`, does it auto-convert to long?

Reply:
INTEGER means non-floating number of any storage size for SQLite: https://www.sqlite.org/datatype3.html